tools/c7n_azure/tests/test_function.py

```diff
@@ -17,7 +17,7 @@
 import unittest
 
 try:
-    from azure.functions_worker.bindings.queue import QueueMessage
+    from azure.functions.queue import QueueMessage
```

Comment:
why is this in a try/except import block in a unit test?

Reply:
nevermind the the ci failure indicates due to not working on python 2.7

Reply:
in general the blanket catch on import errors should be localized to python 2.7 so that the other runtimes actually throw properly on errors.